### ISSUE TITLE
Undefined array key "loopStartDate"

### ISSUE THREAD

**Issue by martinleveille** [percipioglobal/craft-timeloop#53]
I created a Timeloop field, added it to a single, and couldn't edit that entry anymore.

First time using this plugin and it doesn't work.

Timeloop 4.1.0
Craft Pro 4.3.3
PHP 8.0.24
MariaDB 10.6.10

```
yii\base\ErrorException: Undefined array key "loopStartDate" in /var/www/html/vendor/percipiolondon/craft-timeloop/src/fields/TimeloopField.php:260
```

```
public function isValueEmpty(mixed $value, ElementInterface $element): bool
{
    if ($value['loopStartDate'] === []) {
        return parent::isValueEmpty('', $element);
    }

    return false;
}
```

**Comment by mscherr-fastspot** [comment]
I ran into this exact issue as well.

Timeloop 4.1.0
Craft Pro 4.3.10
PHP 8.1.13
MySQL 5.7.39

Confirmed this is also an error on 4.0.0, 4.0.1 and 4.0.2 in my testing.

**Comment by stevefloat** [comment]
Hi there

Seeing the same error.

I should note that when creating a new entry in the section where the Timeloop field was added, everything works as expected. The error occurs when attempting to edit an existing entry (from before the Timeloop field was added). Error as above:

`2023-03-02 16:16:59 [web.ERROR] [yii\base\ErrorException:2] yii\base\ErrorException: Undefined array key "loopStartDate" in /xxx/vendor/percipiolondon/craft-timeloop/src/fields/TimeloopField.php:260`

Any help would be appreciated.

**Comment by mscherr-fastspot** [comment]
Good observation @stevefloat. That's a good distinction I didn't notice before.

**Comment by cookie10codes** [comment]
hi @martinleveille, @mscherr-fastspot and @stevefloat,

Apologies for taking this long to reply to this. Thanks for letting us know about the bug, I've just released 4.1.1 with a bugfix. Let me know if you still have issues
